joblibspark/backend.py

```diff
@@ -131,35 +141,40 @@ def apply_async(self, func, callback=None):
         # Note the `func` args is a batch here. (BatchedCalls type)
         # See joblib.parallel.Parallel._dispatch
 
-        spark_major_minor_ver = \
-            VersionUtils.majorMinorVersion(pyspark.__version__)
-        if spark_major_minor_ver < (3, 2):
-            def run_on_worker_and_fetch_result():
-                # TODO: handle possible spark exception here. # pylint: disable=fixme
```

Comment:
Do we need to do any additional exception handling before removing this TODO?

Reply:
Good catch. I add it back.